hack/copyright.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Copyright 2023 k0s authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+RESULT=0
+FIX=${FIX:=n}
+
+get_year(){
+    # The head -1 has to be added because if a file has been added, removed and added
+    # again we'll get two lines with both dates. We only pick the newest one.
+    git log --diff-filter=A --pretty=format:%as -- ${1} | cut -d- -f1 | head -1 ||
+        (echo "FATAL!! ${1} doesn't seem to be commited in the repo" 1>&2 ; exit 1)
+}
+
+has_copyright(){
+	DATE=${1}
```

Comment:
I prefer using `$...` by default and to fall back to `${...}` only if it's required. WDYT?

Reply:
I don't feel very strongly about this. I will change it.